torch/csrc/autograd/init.cpp

```diff
@@ -351,6 +403,10 @@ static PyMethodDef methods[] = { // NOLINT
   {"set_autocast_enabled", set_autocast_enabled, METH_O, nullptr},
   {"is_autocast_enabled", is_autocast_enabled, METH_NOARGS, nullptr},
   {"clear_autocast_cache", clear_autocast_cache, METH_NOARGS, nullptr},
+  {"set_autocast_cpu_enabled", set_autocast_cpu_enabled, METH_O, nullptr},
```

Comment:
I'd like `set_autocast_enabled` and `is_autocast_enabled` to be renamed correspondingly (ie, to `set_autocast_cuda_enabled` etc). But i can do that myself in a followup PR.

Tbh the question of where these functions should be exposed is generally interesting. I was never sure autograd/init.cpp was the right place, I just put them there because it was easy. Maybe, for example, I should move Python bindings for the cuda versions to somewhere in torch/csrc/cuda, and the Python exposures should look like
```python
torch.cpu.amp.set_autocast_enabled()
torch.cuda.amp.set_autocast_enabled()
```
or maybe we should have a unified
```python
torch.set_autocast_enabled(device_type=X)
```
But right now these functions aren't meant to be user-facing, so I don't think these questions should block the PR.